src/taker/api.rs

```diff
@@ -71,6 +71,8 @@ pub const RECONNECT_SHORT_SLEEP_DELAY: u64 = 10;
 pub const RECONNECT_LONG_SLEEP_DELAY: u64 = 60;
 pub const SHORT_LONG_SLEEP_DELAY_TRANSITION: u32 = 60;
 pub const RECONNECT_ATTEMPT_TIMEOUT_SEC: u64 = 300;
+// TODO: Maker should decide this fee_rate
```

Comment:
This is not actually a feerate but a total. fee. Change the variable name to just `MINER_FEE`. Also add a doc saying that this fee is used for both funding and contract transactions.

Reply:
Done